### ISSUE TITLE
Mouse events on Windows Terminal 1.9 cause bottom to show errors and ignore events

### ISSUE THREAD

**Issue by ClementTsang** [ClementTsang/bottom#562]
Before you report, please take a look at [Troubleshooting](https://clementtsang.github.io/bottom/nightly/troubleshooting) to see if there's already an answer to your problem. Also check out [existing issues](https://github.com/ClementTsang/bottom/issues) and [known problems](https://clementtsang.github.io/bottom/nightly/support/#known-problems) to see if your problem is already reported/known.

Also, **please fill in all fields if possible** - if the issue is too hard to reproduce or vague, it may not be fixed!

## Describe the bug

Give a high level description of the bug.

bottom spits out an error (see below for screenshot) on Windows Terminal when trying to use the mouse, then continues drawing while ignoring events.

## What operating system/architecture are you using?

Please mention what operating system, version, and architecture you're experiencing the problem on (ex: macOS 10.13, x86_64). Please also ensure that your system is [on the list of supported systems](https://github.com/ClementTsang/bottom#support) - bug reports on unsupported systems are likely to be closed.

Windows 10 21H1 19043.1110

## What terminal are you using `bottom` on?

Please mention what terminal/terminal emulator you are using `bottom` on (ex: Konsole, Kitty).

[Windows Terminal v1.9.1942.0](https://github.com/microsoft/terminal/releases/tag/v1.9.1942.0)

## What version are you on?

Please ensure that the bug still exists on the [latest stable release](https://github.com/ClementTsang/bottom/releases/latest) or newer (i.e. nightly). If so, mention
what version you are using here.

v.0.6.3

## How did you install `bottom`?

Please mention how you installed `bottom`. If you manually compiled it, please also mention your _Rust version_.

Through choco

## How can we reproduce this?

Provide detailed steps on _how_ to reproduce your problem, to the best of your ability. Be as detailed as possible, and include config files or flags if relevant.
If maintainers cannot reproduce the bug, it will be very hard to fix it.

1. Open Windows Terminal (v1.9)
2. Open btm
3. Try to use mouse controls and click around

## What is the actual behaviour?

What is the _actual_ result of the given steps? Be detailed. Please include screenshots if the issue is a visual one.

bottom spits out an error as shown in the screenshot when using mouse clicks, then continues drawing, and ignores events.

![image](https://user-images.githubusercontent.com/34804052/126923464-0fcd8d82-ff4e-40c8-ae2d-ee95e02eab58.png)

## What is the expected behaviour?

What do you believe the expected behaviour should be given these steps?

bottom should not show an error like this - if anything, it should have just crashed.

**Comment by ClementTsang** [comment]
Tried updating to 0.20 of crossterm - this issue is not fixed there either.

Barring adding a panic handler, I'll probably just file this upstream.

**Comment by ClementTsang** [comment]
Backtrace:

```bash
thread '<unnamed>' panicked at 'Event flag 3 does not exist.', C:\Users\Clement\.cargo\registry\src\github.com-1ecc6299db9ec823\crossterm_winapi-0.6.2\src\structs\input.rs:200:18
   0: backtrace::backtrace::trace_unsynchronized<closure-0>
             at C:\Users\Clement\.cargo\registry\src\github.com-1ecc6299db9ec823\backtrace-0.3.59\src\backtrace\mod.rs:66   1: backtrace::backtrace::trace<closure-0>
             at C:\Users\Clement\.cargo\registry\src\github.com-1ecc6299db9ec823\backtrace-0.3.59\src\backtrace\mod.rs:53   2: backtrace::capture::Backtrace::create
             at C:\Users\Clement\.cargo\registry\src\github.com-1ecc6299db9ec823\backtrace-0.3.59\src\capture.rs:176
   3: backtrace::capture::Backtrace::new
             at C:\Users\Clement\.cargo\registry\src\github.com-1ecc6299db9ec823\backtrace-0.3.59\src\capture.rs:140
   4: bottom::panic_hook
             at src\lib.rs:285
   5: btm::main::{{closure}}
             at src\bin\main.rs:121
```

**Comment by ClementTsang** [comment]
Long story short rn the state of the problem is that there's an unhandled event flag in crossterm which panics and kills the event handler thread.

I can bandaid this by slapping on a panic handler and trying to recover (I think?), but I'll wait a bit to see if there's a fix to the underlying issue.

**Comment by ClementTsang** [comment]
Note: with https://github.com/crossterm-rs/crossterm-winapi/pull/20 being merged and 0.9 being available, I think(?) this can be unblocked and worked on.
